Documentation/ApiOverview/LinkBrowser/Tutorials/_CustomLinkBrowser/_CustomLinkHandlerJavaScript.rst.txt

```diff
@@ -0,0 +1,46 @@
+.. Generated by https://github.com/TYPO3-Documentation/t3docs-codesnippets 
+.. Extracted from EXT:examples/Resources/Public/JavaScript/github_link_handler.js
+
+.. code-block:: js
+   :caption: EXT:examples/Resources/Public/JavaScript/github_link_handler.js
+
+   /*
+    * This file is part of the TYPO3 CMS project.
```

Comment:
Do we need this full comment?

Reply:
No that is what the todo is for. The codesnippet generator has to be changed so it can also filter these comments from JavaScript